cmd/oceanbench/s/configure.js

```diff
@@ -0,0 +1,19 @@
+function init() {
```

Comment:
This is for the bootstrap form validation: https://getbootstrap.com/docs/5.0/forms/validation/